src/main/java/core/basesyntax/HasArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface HasArea {
+    int area();
```

Comment:
methods naming is much better when it contains verb
```suggestion
    int getArea();
```